src/components/RepoDetail/RepoDetail.js

```diff
@@ -17,11 +18,15 @@ class RepoDetail extends Component {
   }
 
   render() {
-    const { repoDetail } = this.props.location.state;
+    const {
+      location: {
+        state: { repoDetail },
+      },
+    } = this.props;
     return (
       <Container>
-        <Header as="h1" textAlign="center" style={{ padding: '20px' }}>
-          Repo Details: &nbsp;{' '}
+        <Header as="h1" textAlign="center" className={styles.RepoHeader}>
```

Comment:
use camelCase for classNames, as PascalCase is mostly used for Class names

Reply:
Actually, in the course, the instructor was setting PascalCase so I also set that.